app/Http/Controllers/Admin/WorkingScheduleController.php

```diff
@@ -65,44 +65,44 @@ public function chooseWorkplace()
         return view('admin.work_schedules.choose_workspace', compact('workspaces'));
     }
 
-    public function viewByUser($user_id)
+    public function viewByUser($userId)
     {
-        $user = $this->userRepository->findOrFail($user_id);
+        $user = $this->userRepository->findOrFail($userId);
 
         return view('admin.user.timesheet', compact('user'));
     }
 
-    public function getDataUser(Request $request, $user_id)
+    public function getDataUser(Request $request, $userId)
     {
         $this->validate(
             $request,
             [
                 'start' => 'required',
-                'end' => 'required'
+                'end' => 'required',
             ]
         );
         $dates = [
             'start' => $request->start,
             'end' => $request->end
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/WorkingScheduleController.php</br>Each line in an array declaration must end in a comma